### ISSUE TITLE
All results are negative

### ISSUE THREAD

**Issue by sid-roheda** [TimothyHTimothy/FAST-VQA#4]
Attempted to run your implementation via fast_vqa_model.py file in apis. I always get a -0.xx value, which does not match with what is expected (score between 0-100).

**Comment by LahiruJayasinghe** [comment]
@TimothyHTimothy I did transfer learning with this but still getting negative values which are around 0, my labels are 0 and 1. All the predicted values are less than 0.5. So I wonder when we do inference we should do any transformations to align it to some scale. Much appreciate your feedback, thanks

**Comment by teowu** [comment]
Yes. The output scores are correct as during training we only use PLCC and SRCC losses that do not limit the score ranges. This is also the case during fine tuning. 
If you need the absolute scores for a certain range, I would like to recommend you to use the MOS of your target dataset to rescale (a function in finetune.py) your outputs in your training set, and use the parameters for this rescale to guide real inference.
Hope this can help.
